src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  background-color: #50464634;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: auto;
+  background-color: #FFFFFF;
+}
+
+img {
+  display: flex;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+ul {
+  display: flex;
+  margin-right: 30px;
+  list-style-type: none;
+}
+
+li {
+  font-family: Roboto;
+  font-weight: bold;
+  font-size: 12px;
+  margin-right: 20px;
+}
+
+a {
+  text-decoration: none;
+}
+a:hover {
+  color: grey;
+}
+
+.is-active {
+  color: #00ACDC;
+}
+
+.is-active::after {
+  position: relative;
+}
+
+.defaultlink {
+  color: #000000;
+}
+
+.blueobject {
+  width: 37px;
+  height: 4px;
+  position: absolute;
+  top: 56px;
+  border-radius: 8px;
+  background-color: #00ACDC;
+}
```

Comment:
you should implement this line using the after pseudo element
<img width="653" alt="image" src="https://github.com/user-attachments/assets/df28af68-e9ac-474b-989b-4bcb17eb1ccb">